helix-term/src/health.rs

```diff
@@ -134,8 +134,8 @@ pub fn languages_all() -> std::io::Result<()> {
 
     let check_binary = |cmd: Option<String>| match cmd {
         Some(cmd) => match which::which(&cmd) {
-            Ok(_) => column(&cmd, Color::Green),
-            Err(_) => column(&cmd, Color::Red),
+            Ok(_) => column(&format!("✔ {cmd}"), Color::Green),
+            Err(_) => column(&format!("✘ {cmd}"), Color::Red),
```

Comment:
Please use `format("... {}", cmd)` so our MSRV doesn't increase. https://github.com/helix-editor/helix/pull/1881

Reply:
Thank you for the feedback. I have implemented your suggestion.